tests/schedule_test.go

```diff
@@ -1169,3 +1234,14 @@ func (s *ScheduleFunctionalSuite) cleanup(sid string) {
 		})
 	})
 }
+
+func (s *scheduleFunctionalSuiteBase) newContext() context.Context {
+	return testcore.NewContext()
+}
+
+func (s *ScheduleCHASMFunctionalSuite) newContext() context.Context {
+	baseCtx := testcore.NewContext()
+	return metadata.NewOutgoingContext(baseCtx, metadata.Pairs(
+		headers.ExperimentHeaderName, "chasm-scheduler",
+	))
```

Comment:
There should be a way to run the scheduler just with overriding the dynamic config without these specific headers, no?